container/sagemaker/serve.py

```diff
@@ -42,6 +43,16 @@ def __init__(self):
         self._nginx_loglevel = os.environ.get('SAGEMAKER_TFS_NGINX_LOGLEVEL', 'error')
         self._tfs_default_model_name = os.environ.get('SAGEMAKER_TFS_DEFAULT_MODEL_NAME', None)
 
+        self._tfs_enable_batching = os.environ.get('SAGEMAKER_TFS_ENABLE_BATCHING', None)
+        self._tfs_batching_max_batch_size = os.environ.get(
```

Comment:
it might be simpler to not have properties for the batch settings (except enable), and just handle them as locals in _create_batching_config